ui/app/AppLayouts/Wallet/views/AssetsDetailView.qml

```diff
@@ -174,14 +174,15 @@ Item {
                                     label += ': ';
                                 }
                                 label += tooltipItem.yLabel.toFixed(2);
-                                return label.slice(0,label.indexOf(":")+1) + " %1".arg(RootStore.currencyStore.currentCurrencySymbol) + label.slice(label.indexOf(":") + 2, label.length);
+                                return label.slice(0,label.indexOf(":")+1) + " %1".arg(RootStore.currencyStore.currentCurrencySymbol) + label.slice(label.indexOf(":") + 1, label.length);
```

Comment:
This should properly format the currency amount, ie `LocaleUtils.currencyAmountToLocaleString()`

Reply:
Your balance on the left suggests your locale uses a comma `,` (2 116,31 USD) as a decimal separator, yet the video shows the tooltip in the chart formatted as `USD2.01`: both currency symbol placement and the decimal separator being wrong